src/Microsoft.PowerShell.Commands.Diagnostics/CommonUtils.cs

```diff
@@ -14,9 +14,90 @@ namespace Microsoft.PowerShell.Commands.Diagnostics.Common
 {
```

Comment:
@SteveL-MSFT, your last commit had 2 failures in `PowerShell-CI-macos`
ConvertTo-Json.Should handle 'AutomationNull.Value' and 'NullString.Value' correctly
```powershell
Expected strings to be the same, but they were different.
Expected length: 37
Actual length:   33
Strings differ at index 14.
Expected: '{"a":null,"b":null,"c":null,"d":null}'
But was:  '{"a":null,"b":{},"c":null,"d":{}}'
at <ScriptBlock>, /Users/runner/runners/2.160.0/work/1/s/test/powershell/Modules/Microsoft.PowerShell.Utility/ConvertTo-Json.Tests.ps1: line 79
79:         } | ConvertTo-Json -Compress | Should -BeExactly '{"a":null,"b":null,"c":null,"d":null}'
```
ConvertTo-Json.Should handle the ETS properties added to 'DBNull.Value' and 'NullString.Value'
```powershell
Expected strings to be the same, but they were different.
Expected length: 69
Actual length:   56
Strings differ at index 36.
Expected: '..."},{"value...'
But was:  '..."},{"nulls...'
at <ScriptBlock>, /Users/runner/runners/2.160.0/work/1/s/test/powershell/Modules/Microsoft.PowerShell.Utility/ConvertTo-Json.Tests.ps1: line 98
98:             $p1, $p2 | ConvertTo-Json -Compress | Should -BeExactly '[{"value":null,"dbnull":"dbnull"},{"value":null,"nullstr":"nullstr"}]'
```